core/src/main/java/io/questdb/cairo/wal/TableRenameSupportExecutionContext.java

```diff
@@ -38,16 +38,9 @@ class TableRenameSupportExecutionContext extends SqlExecutionContextImpl {
         super(cairoEngine, workerCount, sharedWorkerCount);
     }
 
-    @Override
-    public TableRecordMetadata getMetadata(TableToken tableToken) {
-        final CairoEngine engine = getCairoEngine();
-        return engine.getMetadata(this.tableToken);
-    }
-
-    @Override
-    public TableRecordMetadata getMetadata(TableToken tableToken, long structureVersion) {
+    public TableRecordMetadata getSequencerMetadata(TableToken tableToken) {
         final CairoEngine engine = getCairoEngine();
-        return engine.getMetadata(this.tableToken, structureVersion);
+        return engine.getSequencerMetadata(this.tableToken);
```

Comment:
I think this is used to compile in WAL Apply execution path and should return Reader metadata even when it's asked for for Sequencer metadata.